config/environments/development.rb

```diff
@@ -32,6 +32,10 @@
   config.action_mailer.raise_delivery_errors = false
 
   config.action_mailer.perform_caching = false
+  config.action_mailer.raise_delivery_errors = false
+
+  config.action_mailer.delivery_method = :smtp
+  config.action_mailer.smtp_settings = { address: 'mailcatcher', port: 1025, domain: 'mailcatcher' }
```

Comment:
This change requires folks to map 'mailcatcher` to localhost in `/etc/hosts/` if wanting to see email in mailcatcher. Without doing so, nothing breaks, you just won't be able to verify email sends. This has been added in the README.